.github/workflows/CI.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
```

Comment:
Could you merge PolyChord:master into Ormorod:ormorod-custom_cluster so that the diff is cleaner?

Reply:
It would also be good to add a python test of clustering if you have time.